packages/default-icons/README.md

```diff
@@ -45,3 +45,15 @@ You can choose the way you prefer:
    * SVGR can be run from the [CLI](https://react-svgr.com/docs/cli/)
    * SVGR can be used as a [webpack loader](https://react-svgr.com/docs/webpack/), this way you can import your SVG directly as a React Component
 
+### Export icons from Figma
+Before export icons, you will need to generate a Personal Access Token in Figma.
+
+1. Go to the File Browser.
```

Comment:
File Browser? Like web browser?

Reply:
files in figma

<img width="1792" alt="Screenshot 2022-09-20 at 14 17 27" src="https://user-images.githubusercontent.com/20676525/191244373-76b8368b-5173-4f03-bf02-8ab44572dc87.png">

Reply:
ok, need to change it as it's unclear

Reply:
updated with the link